docs/pipelines/gears.md

```diff
@@ -17,7 +18,10 @@ Pipeline analysis components include:
 
 ## Gears
 
-This pipeline is used to analyse a group of samples. This pipeline only accepts fastq files. The fastq files first get trimmed and clipped with [Flexiprep](Flexiprep). This can be disabled with the config flags of [Flexiprep](Flexiprep). The samples can be specified with a sample config file, see [Config](../general/Config)
+This pipeline is used to analyse a group of samples and only accepts fastq files. The fastq files first get trimmed and clipped with [Flexiprep](flexiprep). This can be disabled with the config flags of [Flexiprep](flexiprep). The samples can be specified with a sample config file, see [Config](../general/config).
+Gears uses centrifuge by default as its classification engine and the NCBI non-redundant database (nt) as the default database against which taxonomic assignments for the short input reads are made.
```

Comment:
The NCBI database is a default on shark, outside shark the database is a required field the user should suply